### ISSUE TITLE
Order of parameters in attach

### ISSUE THREAD

**Issue by s00500** [RoboticsBrno/ESP32-Arduino-Servo-Library#1]
This library changes the order of parameters in the attach() function as compared to the standard arduino library, please change this or document it!
Thanks

**Comment by JarekParal** [comment]
Yes, that is my fault. I'm probably creating a new interface for the function attach() with two options:
1) compatible with Arduino servo library - without parameter channel -> channel will be set automatically
2) parameter channel change from int to enum class and then the standard function overloading will work 

What do you think about this change?

**Comment by s00500** [comment]
yes perfect, I was also thinking about having the channel as an optional attribute :-)

**Comment by JarekParal** [comment]
Actually, I improve the documentation and in the future, I would like to refactor the library with cooperation to #2.
